renderers/react/src/docs/jsxDecorator.tsx

```diff
@@ -122,7 +122,12 @@ export const renderJsx = (code: React.ReactElement, options: JSXOptions) => {
   const result = React.Children.map(code, (c) => {
     // @ts-ignore FIXME: workaround react-element-to-jsx-string
     const child = typeof c === 'number' ? c.toString() : c;
-    let string = applyBeforeRender(reactElementToJSXString(child, opts as Options), options);
+    const toJSXString =
+      typeof reactElementToJSXString === 'function'
+        ? reactElementToJSXString
+        : // @ts-ignore
+          reactElementToJSXString.default;
+    let string = applyBeforeRender(toJSXString(child, opts as Options), options);
```

Comment:
I had to make this change to deal with an inconsistency in `react-element-to-jsx-string`'s exports in different build contexts. I'd suggest we should fix that upstream (also maybe we should maintain that package?)